resources/views/dashboard/getting-started.blade.php

```diff
@@ -1,56 +1,93 @@
+                        )"
+                        :actionLabel="__('Browse organizations')" :actionUrl="localized_route('regulated-organizations.index')" />
+                @elseif($user->individual->isConnector())
+                    <livewire:prompt :model="Auth::user()" prompt="dismissed_browse_organizations_prompt_at"
+                        :heading="__('Find organizations to work with')" :interpretationName="__('Find organizations to work with as a Community Connector', [], 'en')" interpretationNameSpace="getting_started"
+                        :description="__(
+                            'As a Community Connector, you can find organizations that might require your services.',
+                        )" :actionLabel="__('Browse organizations')" :actionUrl="localized_route('regulated-organizations.index')" />
+                @elseif ($user->individual->isParticipant())
+                    <livewire:prompt :model="Auth::user()" prompt="dismissed_browse_engagements_prompt_at"
+                        :heading="__('Find engagements to join')" :interpretationName="__('Find engagements to join', [], 'en')" interpretationNameSpace="getting_started"
+                        :description="__(
+                            'As a participant, you can answer surveys, or join focus groups, workshops, and more.',
+                        )" :actionLabel="__('Browse engagements')" :actionUrl="localized_route('engagements.index')" />
+                @endif
             @endif
-        </div>
-    @else
-        <x-interpretation class="interpretation--center" name="{{ __('Getting started', [], 'en') }}"
-            namespace="getting_started" />
-        <ol class="getting-started__list counter stack" role="list">
-            @if ($user->context === App\Enums\UserContext::Organization->value)
-                @include('dashboard.partials.getting-started-organization')
-            @elseif ($user->context === App\Enums\UserContext::RegulatedOrganization->value)
-                @include('dashboard.partials.getting-started-regulated-organization')
+        @else
+            @if ($user->hasTasksToComplete() && !$user->hasDismissedPrompts())
+                <x-interpretation class="interpretation--center" name="{{ __('Getting started', [], 'en') }}"
+                    namespace="getting_started" />
+
+                <ol class="getting-started__list counter stack" role="list">
+                    @if ($user->context === App\Enums\UserContext::Organization->value)
+                        @include('dashboard.partials.getting-started-organization')
+                    @elseif ($user->context === App\Enums\UserContext::RegulatedOrganization->value)
+                        @include('dashboard.partials.getting-started-regulated-organization')
+                    @endif
+                </ol>
+            @else
+                <p class="h3">{{ __('Congratulations, your role has been fully approved.') }}</p>
             @endif
```

Comment:
With the latest change the Congratulations message is shown instead of the tasks for the FROs and Orgs. A couple options I can think of is to change the `@else` to an `@elseif` to only display when the `$user` has the individual context. Or add what to do next prompts for Orgs and FROs.